### PR TITLE
float: fpclassify wrapper node

### PR DESCRIPTION
**Pull request by edersondisouza**
If one needs to classify a float (that can be NaN, Infinite, Zero,
Subnormal or Normal), this node comes handy.
It receives a float as input and outputs it to the due port.
Tests to this node were added too.

### CODE REVIEW

**Comment by glima** [comment]
It does not hurt to have this, IMO. +1

File: src/modules/flow/float/float.c [hunk 1]
```diff
@@ -962,4 +962,36 @@ wave_generator_sinusoidal_open(struct sol_flow_node *node,
...
+classify_process(struct sol_flow_node *node, void *data, uint16_t port, uint16_t conn_id, const struct sol_flow_packet *packet)
+{
+    double value;
+    int r;
+
+    r = sol_flow_packet_get_drange_value(packet, &value);
+    SOL_INT_CHECK(r, < 0, r);
+
+    switch (fpclassify(value)) {
+        case FP_NAN:
```
Comment (lpereira) [inline src/modules/flow/float/float.c]: Minor coding style issue: please don't indent `case` labels. When in doubt, use the `check-style.sh` script.

**Comment by mbelluzzo** [comment]
+2 with indent fix

**Comment by edersondisouza** [comment]
Integrated with indent fix.

**Pull request closed without merging**